index.js

```diff
@@ -39,16 +39,17 @@ module.exports = file => {
 
 	return Promise.all([
 		mountPoint(userHome),
-		mountPoint(file)
-	]).then(result => {
-		const ret = result[0];
-		const res = result[1];
-
-		if (ret === res) {
+		// ignore errors in case `file` is a dangling symlink
+		// `err &&` is necessary to pass style check
+		mountPoint(file).catch(err => err && Promise.resolve(undefined))
```

Comment:
You can just do: `mountPoint(file).catch(() => {})`

Reply:
Wow, that's so much better – fixed!